src/Microsoft.ML.Data/Transforms/InvertHashUtils.cs

```diff
@@ -41,12 +41,12 @@ private static void ClearDst(ref StringBuilder dst)
             var conv = Conversion.Conversions.Instance;
 
             // First: if not key, then get the standard string converison.
-            if (!type.IsKey)
+            if (!(type is KeyType keyType))
                 return conv.GetStringConversion<T>(type);
 
             bool identity;
             // Second choice: if key, utilize the KeyValues metadata for that key, if it has one and is text.
-            if (schema.HasKeyValues(col, type.KeyCount))
+            if (schema.HasKeyValues(col, keyType.KeyCount))
```

Comment:
>KeyCount [](start = 49, length = 8)

Cool. Could change this to `Count` I think now that you've made this a key type, but not required.

Reply:
I think we can address this when remove KeyCount from the base ColumnType.